lib/diplomat/event.rb

```diff
@@ -151,17 +151,15 @@ def get name=nil, token=:last, not_found=:wait, found=:return
       }
     end
 
-
     private
 
-    def wait_for_next_event url
-      index = @raw.headers["x-consul-index"]
-      url = [url, use_named_parameter("index", index)].join("&")
-      return @conn.get do |req|
+    def wait_for_next_event(url)
+      index = @raw.headers['x-consul-index']
+      url = [url, use_named_parameter('index', index)].join('&')
+      @conn.get do |req|
         req.url concat_url url
-        req.options.timeout = 86400
+        req.options.timeout = 86_400
```

Comment:
This is minor, but I tend to prefer this to be coded as '60 * 60 * 24' so it's clear that its a day.